packages/react-ai/src/components/AIConversation/views/Controls/PromptControl.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import { withBaseElementProps } from '@aws-amplify/ui-react-core/elements';
+
+import {
+  AIConversationElements,
+  ConversationInputContext,
+  MessagesContext,
+  SuggestedPromptsContext,
+} from '../../context';
+import { classNames } from '@aws-amplify/ui';
+import { ControlsContext } from '../../context/ControlsContext';
+
+const { View, Button, Text, Heading, Icon } = AIConversationElements;
+
+const PROMPT_BLOCK = 'ai-prompts';
+const PROMPT_CONTROL = `${PROMPT_BLOCK}__prompt`;
+const PROMPT_CARD = `${PROMPT_CONTROL}__card`;
+
+const PromptCard = withBaseElementProps(Button, {
+  className: PROMPT_CARD,
+  type: 'button',
+});
+
+const AIIconProps = () => ({
+  children: (
+    <>
+      <path
```

Comment:
Think I saw an AI icon declared somewhere else in this PR as well?

Reply:
Don't think so, or at least search doesn't show anything

Reply:
There's a `DEFAULT_AI_ICON` declaration: https://github.com/aws-amplify/amplify-ui/pull/5767/files#diff-cc62d854ca375be87469f7cb4d083bd207b8292bc112c8eb46c7d32710a26aeaR16